pkgs/servers/x11/xorg/default.nix

```diff
@@ -935,22 +935,6 @@ lib.makeScope newScope (self: with self; {
     meta.platforms = lib.platforms.unix;
   }) {};
 
-  # THIS IS A GENERATED FILE.  DO NOT EDIT!
-  libXaw3d = callPackage ({ stdenv, pkg-config, fetchurl, libX11, libXext, libXmu, libXpm, xorgproto, libXt }: stdenv.mkDerivation {
```

Comment:
Almost sure that this is generated by some script, so we probably need to remove the reference for whatever generates it first.

Reply:
It is generated from `pkgs/servers/x11/xorg/tarballs.list`. I generated this file from it.